provisioning/miniprov/hare_mp/main.py

```diff
@@ -395,6 +407,7 @@ class ProcessStartInfo:
     fid: str
 
 
+@func_log(func_enter, func_leave)
 def start_mkfs(proc_to_start: ProcessStartInfo) -> int:
     try:
         logging.debug('Starting mkfs process [fid=%s] at hostname=%s',
```

Comment:
In this latest PR we are not changing .debug to .info @ydb242 ?

Reply:
Because of the logs not being generated after deployment, I had avoided this change in PR.
I will check once again.